java/gradle/ice.gradle

```diff
@@ -55,7 +55,17 @@ slice {
 }
 
 spotless {
+  format 'misc', {
+    // define the files to apply `misc` to
+    target '*.java', '*.gradle'
+
+    // define the steps to apply to those files
+    trimTrailingWhitespace()
+    endWithNewline()
+  }
```

Comment:
Also went ahead and added extra checks for trailing space and newlines.
Our own whitespace script should catch this... But figured why not.